docs.aerogear-site-hosting.md

```diff
@@ -0,0 +1,21 @@
+# Hosting of docs.aerogear.org web site
+
+## Introduction
+
+Previously all AeroGear docs were hosted on a file server and associated with the domain. We'd like to take advantage of github for hosting for reliability and ease of use.
+
+## Problem Description
+
+Hosting html (and associated files) for docs.aerogear.org is essential for communication with users. Need to automatically publish any changes to mobile-docs repo to docs.aerogear.org.
+
+## Expectations
+
+- If we take an approach used by https://github.com/debezium/debezium.github.io where asciidoc and html files are both in same repo (adoc in develop branch, html in master branch), we can anticipate the repo getting large over time with history of html changes mixed with history of adoc file changes.
+
+  
+## Proposed Solution
+
+Create a docs.aerogear.github.io repo and use it to host docs.aerogear.org html files.
```

Comment:
Better to avoid this name as this is githubpages antipattern. 
Best way to do it is to just create any repo name and link that to domain you want like docs.aerogear.org. 
It's really simple and it can be done by just single file in repository. 

docs.aerogear.github.io is used only when you do not have custom domain and it do not support subdomains.

Reply:
@wtrocki  good point, updated